book/src/ui-usage.md

```diff
@@ -1,73 +1,105 @@
 # Usage
+Siren offers many features ranging from diagnostics, logs, validator management including graffiti and exiting. Below we will describe all major features and how to take advantage of Siren to the fullest.
 
-## Dashboard
+# Dashboard
```

Comment:
This needs to be a level-2 heading. In Markdown linter, only 1 top-level heading `#` is allowed in each `.md` file. Reference: https://github.com/DavidAnson/markdownlint/blob/main/doc/md025.md
```suggestion
## Dashboard
```